.yarnrc.yml

```diff
@@ -26,5 +26,8 @@ supportedArchitectures:
   os:
     - darwin
     - linux
+  libc:
+    - current
+    - glibc
```

Comment:
Some packages are built for a specific architecture. We run Terra UI on two types of architectures: developer Macbooks and Linux in CI.

Adding this ensures that the versions of packages required in CI are added to the cache when a developer installs a package on their machine.